### ISSUE TITLE
Puppet readme files out of date

### ISSUE THREAD

**Issue by devrobo** [arista-eosplus/packer-ztpserver#28]
They need to be updated so that puppet-eos can be loaded and used with out issue.

**Comment by devrobo** [comment]
Just found the puppet-ztpserver repo. I think all that needs to be done is to delete the following directories: Fedora/files/puppet and Ubuntu/files/puppet

**Comment by phil-arista** [comment]
Right. I was going to mention the only ref to puppet in the main README was in an old commit. I'll get those stray folders removed.

**Comment by jerearista** [comment]
Should we remove those or should I add in a set of confins/instructions to
just "turn it on"?

That leads to an additional question, though.  We have been releasing much
of our source code as open-source.  However there was some push back about
attaching ready-to-install binary packages to GitHub releases with the
suggestion that anyone getting code from Git should be able to build the
package themselves.      I know we have to draw a line somewhere between
what we do for free vs what we charge for.    What is the current
perspective on that?

In short, when we do a release of rbeapi, do we post a .rpm or .swix file
in the release, on software-downloads, or not at all?   ...When we tell
people how to bootstrap Puppet with ZTP Server, do we tell them to download
the package or build it themselves?

-Jere

On Sun, Aug 9, 2015 at 4:35 PM, Philip DiLeo <notifications@github.com>
wrote:

> Right. I was going to mention the only ref to puppet in the main README
> was in an old commit. I'll get those stray folders removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arista-eosplus/packer-ztpserver/issues/28#issuecomment-129236067>
> .
>



-- 
Jere Julian
Arista Networks <http://arista.com>
EOS+ Advanced Services Engineer
Mobile: 919-250-8217
Office: 408-547-8418
jere@arista.com
